examples/combined_ODNP.py

```diff
@@ -276,56 +243,49 @@
         ini_time = time.time()
         vd_data = run_IR(
             nPoints=nPoints,
-            nEchoes=nEchoes,
+            nEchoes=parser_dict['nEchoes'],
             vd_list_us=vd_list_us,
-            nScans=nScans,
-            adcOffset=adcOffset,
-            carrierFreq_MHz=carrierFreq_MHz,
-            p90_us=p90_us,
-            tau_us=tau_us,
-            repetition=FIR_rd,
-            output_name=output_name,
-            SW_kHz=SW_kHz,
+            nScans=parser_dict['nScans'],
+            adcOffset=parser_dict['adc_offset'],
+            carrierFreq_MHz=parser_dict['carrierFreq_MHz'],
+            p90_us=parser_dict['p90_us'],
+            tau_us=parser_dict['tau_us'],
+            repetition=parser_dict['FIR_rep'],
+            output_name= filename,
+            SW_kHz=parser_dict['SW_kHz'],
             ret_data=None,
         )
         vd_data.set_prop("start_time", ini_time)
         vd_data.set_prop("stop_time", time.time())
-        acq_params = {
-            j: eval(j)
-            for j in dir()
-            if j
-            in [
-                "adcOffset",
-                "carrierFreq_MHz",
-                "amplitude",
-                "nScans",
-                "nEchoes",
-                "p90_us",
-                "deadtime_us",
-                "repetition_us",
-                "SW_kHz",
-                "nPoints",
-                "deblank_us",
-                "tau_us",
-                "MWfreq",
-                "acq_time_ms",
-                "meter_power",
-            ]
-        }
-        vd_data.set_prop("acq_params", acq_params)
-        vd_data.set_prop("postproc_type", IR_postproc)
+        vd_data.set_prop("acq_params", parser_dict.asdict())
+        vd_data.set_prop("postproc_type", "spincore_IR_v1")
         vd_data.name(T1_node_names[j])
-        vd_data.chunk("t", ["ph1", "ph2", "t2"], [len(IR_ph1_cyc), len(IR_ph2_cyc), -1])
+        vd_data.chunk("t", ["ph2", "ph1", "t2"], [len(IR_ph2_cyc), len(IR_ph1_cyc), -1])
         vd_data.setaxis("ph1", IR_ph1_cyc / 4)
         vd_data.setaxis("ph2", IR_ph2_cyc / 4)
-        vd_data.hdf5_write(myfilename)
-    final_frq = p.dip_lock(
-        uw_dip_center_GHz - uw_dip_width_GHz / 2,
-        uw_dip_center_GHz + uw_dip_width_GHz / 2,
-    )
+        vd_data.setaxis('nScans',r_[0:parser_dict['nScans']])
+        nodename = vd_data.name()
+        with h5py.File(
+            os.path.normpath(os.path.join(target_directory,f"{filename_out}")
+        ) as fp:
+            if nodename in fp.keys():
+                print("this nodename already exists, so I will call it temp_%d"%j)
```

Comment:
if it does exist already (IDK why this would happen but just in case), we rename it temp_index so whatever index it was at is labeled and we won't have repeating "temps"